### ISSUE TITLE
predict question of classify task 

### ISSUE THREAD

**Issue by mingmingyang1** [faneshion/MatchZoo#39]
i run the classify task , there is a question that  the accuracy of test in training is as high as 80%, but the accuracy of the test in predict is only 60%. How to deal with it?   

thanks！

**Comment by yangliuy** [comment]
When you did the prediction, which model (specified by 'test_weights_iters' in the config file) did you use? You should choose a correct 'test_weights_iters' to do the prediction. One way is to look at when the accuracy on validation data reaches the highest score.

"global":{
      "model_type": "PY",
      "weights_file": "examples/wikiqa/weights/drmm.wikiqa.weights",
      "save_weights_iters": 10,
      "num_iters": 400,
      "display_interval": 10,
      **"test_weights_iters": 400,**
      "optimizer": "adadelta",
      "learning_rate": 0.0001
  },

**Comment by o0windseed0o** [comment]
@mingmingyang1 @yangliuy  I think we have the same problem, right? 
There are two different accuracies on the same dataset during training and predicting, if you have set the test_weights_iters parameter correctly. The issue is #38
